subprojects/docs/src/docs/release/notes.md

```diff
@@ -135,12 +135,14 @@ Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.
 
 #### Control skipping Kotlin metadata version check for script compilation
 
-You can now control if Kotlin DSL script compilation should skip Kotlin metadata version check.
 Skipping Kotlin metadata version check in Kotlin DSL script compilation is now deprecated.
```

Comment:
Wdyt, about adding a concise explanation (1-2 sentences) about what metadata version check is and when/how/whom it affects?

Reply:
Or at least a link to somewhere having that info.

Reply:
Good point. Added a link